packages/react-router-node/package.json

```diff
@@ -13,8 +13,22 @@
   "license": "MIT",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
```

Comment:
I had to add an ESM build to fix a dual-package-hazard issue. In the `upload-test` integration test we have an `instanceof MaxPartSizeExceededError` check that started failing because the app and `@react-router/node` imported different builds of React Router. It seems the move to using `exports` triggered Playwright to load the ESM build rather than the CJS build.